yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryExecutor.java

```diff
@@ -170,6 +190,62 @@ private Continuation executeContinuation(@Nonnull RelationalConnection connectio
         return continuationAfter;
     }
 
+    private RelationalPreparedStatement prepareContinuationStatement(@Nonnull RelationalConnection connection,
+                                                                     @Nonnull Continuation continuation,
+                                                                     @Nullable Integer maxRows) throws SQLException {
+        var s = connection.prepareStatement("EXECUTE CONTINUATION ?;");
+        s.setMaxRows((maxRows != null) ? maxRows : DEFAULT_MAX_ROWS);
+        if (parameters != null) {
+            setParametersInPreparedStatement(s, connection);
+        }
+        // set continuation
+        s.setBytes(1, continuation.serialize());
+        return s;
+    }
+
+    private Object executeStatementAndForceContinuations(@Nonnull Statement s, @Nullable String queryString,
+                                                         final RelationalConnection connection, @Nullable Integer maxRows) throws SQLException {
+        // Check if we need to force continuations
+        if ((maxRows == null) && forceContinuations) {
+            s.setMaxRows(FORCED_MAX_ROWS);
+            Object result = executeStatement(s, queryString);
+            if (result instanceof RelationalResultSet) {
```

Comment:
This method is rather complex, perhaps a method for the contents of this `if` would be helpful.

Reply:
Done.